cegs_portal/search/views/v1/dna_features.py

```diff
@@ -75,6 +75,12 @@ def get(self, request, options, data, id_type, feature_id):
 
             feature_reos.append((feature, sources, targets))
 
+        non_targeting_reos = []
+        for feature in data.all():
+            reos = DNAFeatureSearch.non_targeting_reo_search(feature.pk)
```

Comment:
I think it would be better api design to pass the whole feature into the `non_targeting_reo_search` function and then that function can call `gene.pk`.

Reply:
Also, `x.pk` isn't used anywhere else in the codebase. Does `gene.id` not work?

Reply:
Got it. Gene Id is not working for some reason. "x.pk" seemed to resolve the error from gene.id.